### ISSUE TITLE
PHP Strict Standards

### ISSUE THREAD

**Issue by jpswade** [pear/Net_SMTP#12]
```
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 448
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 465
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 472
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 515
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 263
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 519
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 597
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 263
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 600
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 603
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 515
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 263
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 519
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 621
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 802
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 263
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 806
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 814
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 263
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 818
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 822
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 263
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 827
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 649
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 945
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 263
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 948
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 975
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 263
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 978
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 1054
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 263
PHP Strict Standards:  Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /.../Net/SMTP.php on line 1057
```

**Comment by alecpl** [comment]
In recent PHP versions this is E_DEPRECATED. Please, fix this.

**Comment by lbltls** [comment]
This bug is currently filling our logs in dimensions of Gigabytes... Until this finally gets fixed, the only solution is to disable logging of E_DEPRECATED which is not a good solution, however.
Please fix this asap, thank you.

Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Mail/smtp.php on line 365
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 448
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 465
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 472
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 515
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 263
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 519
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Mail/smtp.php on line 249
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Mail/smtp.php on line 285
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 945
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 948
Deprecated: Non-static method Mail_RFC822::parseAddressList() should not be called statically, assuming $this from incompatible context in /usr/share/php/Mail.php on line 253
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 975
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 978
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 1054
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 1057
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 1117
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 1127
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 1132
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 489
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 492
Deprecated: Non-static method PEAR::isError() should not be called statically, assuming $this from incompatible context in /usr/share/php/Net/SMTP.php on line 495

**Comment by jparise** [comment]
I believe this has been addressed by #15.

**Comment by alecpl** [comment]
No, but it has been fixed in PEAR-1.10.0 (no stable release yet).

**Comment by jparise** [comment]
More information on upgrading to PEAR-1.10 (dev) is available here:

http://cweiske.de/tagebuch/pear-1.10.0dev1.htm
